sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_shared/_polling.py

```diff
@@ -33,11 +33,11 @@ class KeyVaultOperationPoller(LROPoller):
     # pylint: disable=arguments-differ
     def __init__(self, polling_method):
         # type: (PollingMethod) -> None
-        super(KeyVaultOperationPoller, self).__init__(None, None, None, NoPolling())
+        super(KeyVaultOperationPoller, self).__init__(None, None, self.result, NoPolling())
```

Comment:
Was there a mypy complaint about passing `None` here? I don't think `self.result` would be a fitting deserialization callback here -- the `deserialization_callback` provided to LROPoller should accept a Response as input and return a deserialized object ([link](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/polling/_poller.py#L151)).

Reply:
This was the error:
`azure\keyvault\secrets\_shared\_polling.py:36: error: Argument 3 to "__init__" of "LROPoller" has incompatible type "None"; expected "Callable[..., Any]"`

Reply:
Ah, makes sense. I would either add a `type: ignore` comment here or in that case provide a callable that returns `None`; for example, `lambda *_: None`:
```python
super(KeyVaultOperationPoller, self).__init__(None, None, lambda *_: None, NoPolling())
```